judoscale-ruby/lib/judoscale/config.rb

```diff
@@ -87,6 +87,7 @@ def reset
       @log_tag = "Judoscale"
       @max_request_size_bytes = 100_000 # ignore request payloads over 100k since they skew the queue times
       @report_interval_seconds = 10
+      @rake_task_ignore_regex = /assets:|db:/
```

Comment:
This might be more Rails specific than Ruby specific, I guess, since it's only something we use there... maybe we should only expose it within Rails, like we have `start_reporter_after_initialize`? https://github.com/judoscale/judoscale-ruby/blob/92a34b90eeab819db83b4ee78496638a7cc1c198/judoscale-rails/lib/judoscale/rails/config.rb#L4-L6

(btw, it does make me wonder if we should have `config.rails.foo` like we have with the background workers, but that's probably another topic for discussion)

Reply:
Good call! I'm moving this to be Rails-specific.